.travis.yml

```diff
@@ -43,6 +73,11 @@ before_script:
   - python ci/install_coverage_subprocess_pth.py
 script:
   - COVERAGE_PROCESS_START="$TRAVIS_BUILD_DIR/.coveragerc" PYTHONPATH='.:tests' pytest -r s
+  - if [[ $TRAVIS_EVENT_TYPE == "cron" ]]; then
```

Comment:
Same comment:

instead of testing on `$TRAVIS_EVENT_TYPE` can you please test on `[[ "$PROJECT" != "" ]]`